runtime/src/chpl-cache.c

```diff
@@ -3675,7 +3675,7 @@ void chpl_cache_invalidate(c_nodeid_t node, void* raddr, size_t size,
   TRACE_PRINT(("%d: task %d in chpl_cache_invalidate %s:%d %d bytes at %d:%p\n",
                chpl_nodeID, (int)chpl_task_getId(),
                chpl_lookupFilename(fn), ln,
-               (int)size, node, raddr, addr));
+               (int)size, node, raddr));
```

Comment:
Why do we have all these cache changes here? Are they leftovers from a stale branch or something?

If they are related to the main purpose of this PR somehow, could you mention that in the PR message?

Reply:
I updated the PR message to say

> While I was investigating, I discovered that the verbose comm output with `--cache-remote` enabled was not as useful as it could be. So, this PR makes a couple of updates to fix verbose comm output with the cache enabled.

This is a "while there" change for this PR. It was part of the effort to try to understand the comms (which caused me to reduce some of them and make this PR). I think it is relatively small but if you need me to I can pull it out into another PR.

Reply:
It can stay in this PR and I am fine with the PR message change, thanks!